core/ecschema-editing/src/Differencing/SchemaDiagnosticVisitor.ts

```diff
@@ -254,20 +253,20 @@ export class SchemaDiagnosticVisitor {
     const [enumerator] = diagnostic.messageArgs as [AnyEnumerator];
     this.addEntry({
       changeType: "add",
-      schemaType: "Enumeration",
+      schemaType: SchemaItemTypeName.Enumeration,
       itemName: enumeration.name,
       path: "$enumerators",
-      json: enumerator,
+      difference: enumerator,
     });
   }
 
   private lookupEnumeratorEntry(changeType: DifferenceType, item: string, enumeratorName: string) {
-    return this._differenceReport.changes && this._differenceReport.changes.find((change) => {
+    return this._changes && this._changes.find((change: any) => {
```

Comment:
same here why make type explicitly any?

Reply:
btw I get that the union type here does not have itemName but I think that's an indication that there is something wrong rather than an indication that you should cast away the type